pkg/crc/machine/delete.go

```diff
@@ -39,5 +44,55 @@ func (client *client) Delete() error {
 			logging.Warnf("Failed to remove crc contexts from kubeconfig: %v", err)
 		}
 	}
+	return removeCRCHostEntriesFromKnownHosts()
+}
+
+func removeCRCHostEntriesFromKnownHosts() error {
+	knownHostsPath := filepath.Join(constants.GetHomeDir(), ".ssh", "known_hosts")
```

Comment:
Fwiw, I would have added this as a generic helper in `pkg/crc/ssh`, `ssh.RemoveIPFromKnownHosts("192.168.130.11")` or such. This can be changed later if desired.